### PR TITLE
[Issue 5808][documentation]fix example code

### LINKED ISSUE #5808: [docs] The example code does not display correctly
**Describe the bug**
Ununderstood example code .
**To Reproduce**
1. Go to 'http://pulsar.apache.org/docs/en/schema-understand'
2. Click on 'Understand schema'
3. Scroll down to 'Complex type>Example'

**Expected behavior**
this  example code is for java or other?
**Screenshots**
图片

**Additional context**
For JAVA , ``` Schema> ```  means ``` Schema<KeyValue<Integer, String>>```,right?

### PR DESCRIPTION
**Pull request by huangdx0726**
fix example code

Fixes #5808 

### Motivation

Example code display error.

### Modifications

remove text from ```text

### CODE REVIEW

**Comment by wolfstudy** [comment]
@Anonymitaet PTAL

File: site2/docs/schema-understand.md [hunk 1]
```diff
@@ -220,7 +220,7 @@ This example shows how to construct a key/value schema and then use it to produc
 
 1. Construct a key/value schema with `INLINE` encoding type.
 
-    ```text
+    ```
```
Comment (codelipenghui) [inline site2/docs/schema-understand.md]: If the example is using java,  i recommend using  <code>``` java</code>  here, keywords can be highlighted

Comment (huangdx0726) [reply site2/docs/schema-understand.md]: Ok,I'll fix it.

Comment (huangdx0726) [reply site2/docs/schema-understand.md]: I'v fixed,PTAL.

**Comment by codelipenghui** [comment]
retest this please

**Comment by codelipenghui** [comment]
run integration tests

**Comment by jiazhai** [comment]
run integration tests

**Pull request merged**